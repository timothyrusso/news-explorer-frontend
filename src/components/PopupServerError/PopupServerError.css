.popup-server-error {
  font-family: 'Inter', Arial, sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 12px;
  line-height: 1;
  margin: 0;
  color: #FF0000;
  display: flex;
  justify-content: center;
}
