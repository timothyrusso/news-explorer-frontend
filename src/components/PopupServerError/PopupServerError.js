import "./PopupServerError.css";

const PopupServerError = ({ popupServerError }) => {
  return <p className="popup-server-error">{popupServerError}</p>;
};

export default PopupServerError;
