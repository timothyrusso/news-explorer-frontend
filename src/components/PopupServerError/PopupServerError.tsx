import React, { FC } from 'react';
import './PopupServerError.css';

type PopupServerErrorProps = {
  popupServerErrorMessage: string;
};

const PopupServerError: FC<PopupServerErrorProps> = ({
  popupServerErrorMessage,
}) => {
  return <p className="popup-server-error">{popupServerErrorMessage}</p>;
};

export default PopupServerError;
