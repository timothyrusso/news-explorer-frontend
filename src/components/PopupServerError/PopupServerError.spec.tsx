import React from 'react';
import { render } from '@testing-library/react';
import PopupServerError from './PopupServerError';

describe('PopupServerError component', () => {
  it('Should render the PopupServerError component', () => {
    const { asFragment } = render(
      <PopupServerError popupServerErrorMessage="Message" />
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
