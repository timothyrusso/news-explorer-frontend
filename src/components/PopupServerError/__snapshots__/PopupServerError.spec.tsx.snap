// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PopupServerError component Should render the PopupServerError component 1`] = `
<DocumentFragment>
  <p
    aria-live="polite"
    class="popup-server-error"
  >
    Message
  </p>
</DocumentFragment>
`;
